test/e2e/kubectl.go

```diff
@@ -238,6 +240,44 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		var nsFlag string
+		var rcName string
+		AfterEach(func() {
+			runKubectl("stop", "rc", rcName, nsFlag)
```

Comment:
Sorry, it seems I've lead you astray.  This doesn't do what you want because at this point, rcName and nsFlag are unassigned, so this is effectively scheduling a call to `kubectl stop rc '' ''`.

Please disregard my earlier comment, apparently even simple string initialization needs to be in a BeforeEach. Ginkgo has weird behavior when you try to initialize variables inside of a `Describe`.